CHANGES.md

```diff
@@ -10,6 +10,21 @@
 
 [//]: # (towncrier release notes start)
 
+## 0.4.0 (2024-04-30)
+
+
+
+No significant changes.
```

Comment:
Do you want to mention the compatibility bump?

Reply:
Not a bad idea.

Reply:
Does this suffice?